docs/content/docs/data-structures/counter.md

```diff
@@ -1,5 +1,196 @@
 ---
 title: SharedCounter
-menuPosition: 9
-status: unwritten
+menuPosition: 5
 ---
+
+## Introduction
+
+The `SharedCounter` distributed data structure (DDS) is used to store an integer counter value that can be modified collaboratively.
```

Comment:
I prefer "by multiple clients simultaneously" to "collaboratively". I think we should try to avoid "collab" in this context completely. IMO collab is something humans do, while editing data is something the computer does (perhaps on behalf of a human).

Reply:
I can see your point. And I've updated the text.

I'm still new enough to the space that I'm not sure I have a strong opinion here, but one potential counterargument that occurred to me is that potential customers evaluating our tech may be thinking in terms of human-driven application scenarios. @Rick-Kirkham you've been working through the persona-driven aspects of the site content - what's your take on this?

Reply:
It's true that they are thinking of collab scenarios when first investigating Fluid. But I think that someone who's reading a deep dive about a particular DDS is past that and is now thinking in terms of what the computer or code is doing, so I'd vote with @tylerbutler.